src/plonemeeting/portal/core/content/institution.py

```
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-
+from Acquisition import aq_base
```

Flake8 (7.3.0)

[error] 'Acquisition.aq_base' imported but unused

(F401)